rogueutil.h

```diff
@@ -266,7 +266,7 @@ typedef enum key_code {
 	KEY_NUMPAD6 = 132,
 	KEY_NUMPAD7 = 133,
 	KEY_NUMPAD8 = 134,
-	KEY_NUMPAD9 = 135,
```

Comment:
It's not covered in the style guidelines, but I don't necessarily have a stylistic problem with this as it makes it easier to add new values. I suppose it doesn't matter either way.

Reply:
Whichever is fine by me. C89 dislikes it, which is why I added it in, but if you don't care then I understand.

Reply:
Fair point on C89 compatibility.
Feel free to leave the comma out. :)